### PR TITLE
Server 1256

### PR DESCRIPTION
**Pull request by soulchips**
# Description
We're improving the section on preflight checks to make it a bit more obvious on what the checks are.

# Reasons
Currently, the section on preflight checks is a bit hidden, appearing to be a part of the nomad MTLS section. This change clearly separated the preflight checks as another step and what to expect from this step

### CODE REVIEW

**Review by lloydChris — APPROVED** [review]

**Review by ryanwohara — APPROVED** [review]

File: jekyll/_cci2/server-3-install.adoc [hunk 1]
```diff
@@ -543,13 +543,23 @@ This section can be left with its default values until <<Step 3 - Obtain Load Ba
 #### Enable Mutual TLS (mTLS)
 mTLS encrypts and authenticates traffic between your Nomad control plane and Nomad clients. You should disable mTLS until you have completed <<Step 4 - Install Nomad Clients>> and can obtain the certificate, private key and certificate authority output after completing Step 4. 
 
+#### Preflight Checks
 When all required information has been provided, click the *Continue* button and your CircleCI installation will be put
-through a set of preflight checks to verify your cluster meets the minimum requirements and attempt to deploy. When completed successfully,
-you should see something like the following and you can continue to the next step:
+through a set of preflight checks to verify your cluster meets the minimum requirements and attempt to deploy. Currently we check for
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-install.adoc:548]: Just realised "and attempt to deploy" doesn't make sense at the end of this sentence. Should it be "... minimum requirements for deployment." or something?

File: jekyll/_cci2/server-3-install.adoc [hunk 2]
```diff
@@ -543,13 +543,23 @@ This section can be left with its default values until <<Step 3 - Obtain Load Ba
...
+* The cluster has 30Gi of total memory or greater
+* Total vCPUs in the cluster is 8 or greater
+* The cluster has a default storage class
+
+When completed successfully, you should see something like the following and you can continue to the next step:
 
 .Sever v{serverversion} Preflight Checks
 image::preflight-checks.png[Preflight Checks]
 
+However, should any of these checks fail the issue will be highlighted, giving you an opportunity to address these issues before deployment.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-install.adoc:561]: Can we move this sentence up to the paragraph above the screenshot? so:

"When completed successfully, you should see something like the following and you can continue to the next step. If any checks fail the relevant issue will be highlighted so you can address any issues before deployment."

File: jekyll/_cci2/server-3-install.adoc [hunk 3]
```diff
@@ -543,13 +543,23 @@ This section can be left with its default values until <<Step 3 - Obtain Load Ba
...
+the following:
+
+* Kubernetes Version is 1.16.0 or greater
+* The cluster has 30Gi of total memory or greater
+* Total vCPUs in the cluster is 8 or greater
+* The cluster has a default storage class
+
+When completed successfully, you should see something like the following and you can continue to the next step:
 
 .Sever v{serverversion} Preflight Checks
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-install.adoc:558]: I know this isn't part of the changes but Sever need changing to Server here if possible

**Review by rosieyohannan — COMMENTED** [review]
Thank you! Just a couple of comments then good to go :-D

**Pull request merged**